### ISSUE TITLE
FillType name

### ISSUE THREAD

**Issue by TheAshladzz** [modelleicher/FS22_realismAddon_fillables#22]
Hello! :) 

Maybe not a direct issue, but more curiosity! I am trying to use some mods that shows the filltype name on the fillType level indicator, but not really getting it to work. So im wondering if its a compability issue with this mod? 
Ddoes this mod also draw the whole fillType level indicator on the screen?

**Comment by KITT3000** [comment]
No, it only adds a line with its text in a certain position, it does not even refer to the position of the game level bar only the coordinates are rigidly fixed.
